Python/instrumentation.c

```diff
@@ -566,7 +566,13 @@ de_instrument(PyCodeObject *code, int i, int event)
     _Py_CODEUNIT *instr = &_PyCode_CODE(code)[i];
     uint8_t *opcode_ptr = &instr->op.code;
     int opcode = *opcode_ptr;
-    assert(opcode != ENTER_EXECUTOR);
```

Comment:
This assertion is correct. Please don't remove assertions, unless you are really sure that they are incorrect.

`ENTER_EXECUTOR` should never have associated instrumentation.

Reply:
It is moved to L574, it will be same effect no?